app/components/toolbelt.js

```diff
@@ -11,12 +11,16 @@ class Toolbelt extends Component {
       btnClass = 'btn-success';
     }
     return (
-      <div className="col-md-10 col-md-offset-1">
-        <Button value="Handshake" onClick={this.props.handshake}/>
-        <button className={`btn ${btnClass}`} data-status={cpuProfile.status} onClick={this.props.handleCpuSampling}>
-          {btnText}
-          {cpuProfile.status === 'RUNNING' ? <i className="fa fa-spinner fa-spin"></i> : null}
-        </button>
+      <div className="row">
+        <div className="col-md-2">
+          <Button value="Handshake" onClick={this.props.handshake}/>
+        </div>
+        <div className="col-md-3">
+          <button className={`btn ${btnClass}`} data-status={cpuProfile.status} onClick={this.props.handleCpuSampling}>
```

Comment:
This is minor, but I found returning the entire classname in the classname generating routine pretty helpful in making the JSX cleaner. For instance, the `btnClass` can hold `'btn btn-success` thereby removing the use of interpolation here.